modules/administration/pages/organizations.adoc

```diff
@@ -1,141 +1,88 @@
-This has the advantage that the package installation action will take place immediately, when the schedule is actually executed.
-This "`pre-fetching`" saves maintenance window time, which is good for service uptime.
+=== Organization Users
 
-For staging contents ("`pre-fetching`"), edit on the server [path]``/etc/sysconfig/rhn/up2date``:
+Navigate to the [guimenu]``Users`` tab to view the list of all users associated with the organization, and their role.
+Clicking on a username takes you to the [guimenu]``Users`` menu to add, change, or delete users.
 
-----
-stagingContent=1
-stagingContentWindow=24
-----
 
-`stagingContentWindow` is a time value expressed in hours and determines when downloading will start.
-It is the number of hours before the scheduled installation or update time.
-In this case, it means `24` hours before the installation time.
-The start time for download depends on the selected contact method for a system.
-The assigned contact method sets the time for when the next [command]``mgr_check`` will be executed.
 
-Next time an action is scheduled, packages will automatically be downloaded but not installed yet.
-When the scheduled time comes, the action will use the staged version.
+=== Trusted Organizations
 
-Minion Content Staging::
-Every Organization administrator can enable Content Staging from the Organization configuration page menu:Admin[Organization > OrgName > Configuration > Enable Staging Contents].
+Navigate to the [guimenu]``Trusts`` tab to add or remove trusted organizations.
+Establishing trust between organizations allow them to share content more easily between them, and gives you the ability to migrate clients from one organization to another.
 
-Staging content for clients is affected by two parameters.
 
-* [path]``salt_content_staging_advance:`` expresses the advance time, in hours, for the content staging window to open with regard to the scheduled installation/upgrade time.
-* [path]``salt_content_staging_window:`` expresses the duration, in hours, of the time window for Salt clients to stage packages in advance of scheduled installations or upgrades.
 
-A value of *salt_content_staging_advance* equal to *salt_content_staging_window* results in the content staging window closing exactly when the installation/upgrade is scheduled to be executed.
-A larger value allows separating download time from the installation time.
+=== Configure Organizations
 
-These options are configured in [path]``/usr/share/rhn/config-defaults/rhn_java.conf``
- and by default assume the following values:
+Navigate to the [guimenu]``Configuration`` tab to manage the configuration of your organization.
+This includes the use of staged contents, setting up crash reporting, and the use of of SCAP files.
 
-* [path]``salt_content_staging_advance: 8 hours``
-* [path]``salt_content_staging_window: 8 hours``
+For more information about content staging, see xref:administration:content-staging.adoc[].
```

Comment:
If there are two or more items, we'd back make this an itemized list.

Reply:
I put it into prose instead, it's a pretty complicated concept.